src/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { useNotesContext } from "../../context";
+import { Card } from "../Card/Card";
+import styles from "./sidebar.module.css";
+import { MdStickyNote2 } from "react-icons/md";
+import { FaFilter } from "react-icons/fa";
```

Comment:
Suggestion:  you can import all icons you are using in a single index.js file inside an icon or assets folder and then import them in a single line in the component that you want to use them, so you do not have to write multiple statements for importing different icons.

Reply:
Will do that.